easybuild/easyconfigs/m/makeinfo/makeinfo-7.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'makeinfo'
+version = '7.0'
```

Comment:
There is a newer version available: `7.0.3`. Could we update to the latest version here?

Reply:
my bad, fixed it